components/engine_traits/src/engine.rs

```diff
@@ -263,6 +274,14 @@ pub trait TabletFactory<EK>: TabletAccessor<EK> + Send + Sync {
     /// Get the tablet path by id and suffix
     fn tablet_path(&self, id: u64, suffix: u64) -> PathBuf;
 
+    /// Get the tablet path by id and suffix with "split_" prefix in it.
+    ///
+    /// Only used during split execution.
+    fn split_temp_tablet_path(&self, _id: u64, _suffix: u64) -> PathBuf {
```

Comment:
Instead of using an explict method, I suggest to change existing `tablet_path` and make it accept a custom prefix.

Reply:
Except split, is there any situations that we need customized tablet path?

Reply:
Merge will also create a temporary tablet.